docs/docs/guides/groups.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Creating a Group
+---
+
+# Creating a group in Infrahub
+
+Within Infrahub a [Group](/topics/groups) is defined through the UI and allow you to create and manage collections of objects.
```

Comment:
This sentence is weird. We say a group is defined in the UI, but then use GraphQL instead. A group can be created in multiple ways: in the UI, GraphQL, SDK,